code/modules/tgui_input/checkboxes.dm

```diff
@@ -14,13 +14,17 @@
 	if (!user)
 		user = usr
 	if(!length(items))
-		return
+		return null
```

Comment:
whats the purpose of doing this? is this just more explicit?

Reply:
yes it's more explicit.